SpatialGDK/Documentation/content/schema.md

```diff
@@ -0,0 +1,47 @@
+# Schema
+
+SpatialOS uses [schema (SpatialOS documentation)](https://docs.improbable.io/reference/13.6/shared/concepts/schema#schema) to generate APIs specific to the components in your game world. These APIs define how you can operate on entity components within SpatialOS. 
+
+## Generating schema
+
+To generate schema, select the **Schema** button in the [GDK Toolbar]({{urlRoot}}/content/toolbars#buttons). The GDK automatically iterates through objects with replicated properties to generate the required schema files and then updates the [`SchemaDatabase`(Unreal GDK Glossary)]({{urlRoot}}/content/glossary#schemadatabase).
+
+![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/schema-button.png)
+
+*Image: In the GDK toolbar in the Unreal Editor, select the* ***Schema*** *button.*
+
+As the GDK automatically generates all the schema you need, you do not have to write or edit schema manually when using the GDK.
+
+### When to generate schema
+
+You must generate schema:
+
+- When you add or change any replicated properties that you want to deploy to SpatialOS.
+- When you create a new map. This ensures that you generate schema for any unique objects on this map. 
+- When you open a map and you haven’t generated schema for it yet. 
+
+You must also generate a snapshot after generating schema by selecting the **Snapshot** button on the GDK toolbar. For more information about snapshots, refer to the [snapshot documentation]({{urlRoot}}/content/generating-a-snapshot).
```

Comment:
This is also not true now that generated snapshots don't contain any level information,

Reply:
Also nuked this bit.